recipes/msmpi-sdk/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version="7.1.12437.25" %}
+
+package:
+  name: msmpi-sdk
+  version: {{ version }}
+
+source:
+  fn: msmpisdk.msi
+  url: https://download.microsoft.com/download/E/8/A/E8A080AF-040D-43FF-97B4-065D4F220301/msmpisdk.msi
+  sha256: d111e033df14e61acc9b2cf473e4348f118b89abe730b9b580a43d4b7361a37c
+
+build:
+  number: 0
+  skip: True  # [not win]
+
+test:
+  commands:
+    - copy "C:\Program Files%x86_DIR%\Microsoft SDKs\MPI\Lib\%ARCH_DIR%\*" %LIBRARY_LIB%\
+    - copy "C:\Program Files%x86_DIR%\Microsoft SDKs\MPI\Include\*" %LIBRARY_INC%\
+    - copy "C:\Program Files%x86_DIR%\Microsoft SDKs\MPI\Include\%ARCH_DIR%\*" %LIBRARY_INC%\
+    - copy "C:\Program Files%x86_DIR%\Microsoft SDKs\MPI\License\license_sdk.rtf" %SRC_DIR%\
+
+
+about:
+  summary: Microsoft's MPI (message passing interface) implementation
+  home: https://msdn.microsoft.com/en-us/library/windows/desktop/bb524831(v=vs.85).aspx
+  license: Proprietary
```

Comment:
Not sure.  Maybe not:

```
a.	Distributable Code. The software contains code that you are permitted to distribute in programs you develop if you comply with the terms below.
i.	Right to Use and Distribute. 
•	You may copy and distribute the object code form of the code as part of your MPI applications.
•	Third Party Distribution. You may permit distributors of your programs to copy and distribute the Distributable Code as part of those applications.
ii.	Distribution Requirements. For any Distributable Code you distribute, you must
•	require distributors and external end users to agree to terms that protect it at least as much as this agreement; 
•	display your valid copyright notice on your MPI applications; and
•	indemnify, defend, and hold harmless Microsoft from any claims, including attorneys’ fees, related to the distribution or use of your MPI applications.
iii.	Distribution Restrictions. You may not
•	alter any copyright, trademark or patent notice in the Distributable Code;
•	use Microsoft’s trademarks in your programs’ names or in a way that suggests your programs come from or are endorsed by Microsoft;
•	distribute Distributable Code to run on a platform other than the Windows platform;
•	include Distributable Code in malicious, deceptive or unlawful programs; or
•	modify or distribute the source code of any Distributable Code so that any part of it becomes subject to an Excluded License. An Excluded License is one that requires, as a condition of use, modification or distribution, that
•	the code be disclosed or distributed in source code form; or
•	others have the right to modify it.
```

I can change the recipe so that it just downloads the installer and runs it with a post-link script, I guess.